playbooks/roles/streisand-gateway/templates/vhost.j2

```diff
@@ -1,31 +1,52 @@
 # SSL Gateway
 server {
-  listen 127.0.0.1:{{ nginx_port }} default_server ssl;
+  listen 127.0.0.1:{{ nginx_port }} default_server ssl http2;
 
 {% if le_ok %}
-  ssl_certificate      {{ le_certificate }};
-  ssl_certificate_key  {{ le_private_key }};
+  ssl_certificate         {{ le_certificate }};
+  ssl_certificate_key     {{ le_private_key }};
+  ssl_trusted_certificate {{ le_chain }};
```

Comment:
`le_chain` isn't defined - I think you should close https://github.com/StreisandEffect/streisand/pull/1028 and add that commit to this PR so everything is self-contained.